dnf-behave-tests/features/microdnf/install-xml-base.feature

```diff
@@ -0,0 +1,100 @@
+@no_installroot
+@destructive
```

Comment:
The `@no_installroot` decorator itself is listed among destructive tags, you do not add `@destructive`